Welolo/client/src/components/SHOW_MERCHANT_MENU.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import "../styles/main.css"
+import { getMerchantData } from '../util/getMerchantData';
+import { withRouter } from '../util/withRouter';
+
+class ND_SHOW_MERCHANT_MENU extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: null,
+            items_selected: ''
```

Comment:
I'm getting this error when running it:

```
Error: Failed validation
    at emergepaySdk.<anonymous> (C:\Users\alyss\Documents\School\CS 481\Projects\f21-welolo\Welolo\node_modules\emergepay-sdk\dist\emergepaySdk.js:126:31)
    at step (C:\Users\alyss\Documents\School\CS 481\Projects\f21-welolo\Welolo\node_modules\emergepay-sdk\dist\emergepaySdk.js:44:23)
    at Object.throw (C:\Users\alyss\Documents\School\CS 481\Projects\f21-welolo\Welolo\node_modules\emergepay-sdk\dist\emergepaySdk.js:25:53)
    at rejected (C:\Users\alyss\Documents\School\CS 481\Projects\f21-welolo\Welolo\node_modules\emergepay-sdk\dist\emergepaySdk.js:17:65)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

This was a bug I ended up fixing. It was due to an issue with the types being passed in. Once I saved it to state it fixed it. You may also need to use == rather than === if you ever compare state with the value passed in as they were different types when I worked on it. I used this site to help narrow down the issue https://dev.gravitypayments.com/docs/emergepay/common-error-codes/tip-adjust-error-codes/